openforcefield/topology/molecule.py

```diff
@@ -3490,16 +3490,23 @@ def chemical_environment_matches(
 
         return matches
 
-    # TODO: Move OE-dependent parts of this to toolkits.py
     @classmethod
     @OpenEyeToolkitWrapper.requires_toolkit()
```

Comment:
```suggestion

```

We should be able to ditch this now that it routes to a ToolkitRegistry. Trying to call the method will result in an informative ToolkitRegistry error like "I tried to do `from_iupac` but I only know about RDKitToolkitWrapper and AmberToolsToolkitWrapper and neither of them can do `to_iupac`"